### DISCUSSION TITLE
Deep Learning using River

### DISCUSSION THREAD

**Discussion by Nirmal-Prabhu** [online-ml/river#841] [Q&A] [answered]
Thank you to the river team for this easy to use intuitive library. I was wondering if and how I could implement Deep Learning using River.  There's clear documentation and examples of Decision trees and other commonly used machine learning techniques but I was hoping for some information on online incremental Deep Learning.

**Comment by MaxHalford** [comment, marked as answer]
Hello there! There's a vanilla MLP regressor you can find [here](https://riverml.xyz/latest/api/neural-net/MLPRegressor/). There's also a `PyTorch2RiverClassifier` you can find [here](https://riverml.xyz/latest/api/compat/PyTorch2RiverClassifier/) and a `PyTorch2RiverRegressor` you'll find [here](https://riverml.xyz/latest/api/compat/PyTorch2RiverRegressor/). But that's it really, we haven't used this stuff a lot.
